src/main/webapp/app/shared/metis/metis.component.scss

```diff
@@ -25,6 +25,10 @@
         font-weight: 500;
         padding-right: 5px;
     }
+
+    .posting-author-role {
+        font-size: 17px;
```

Comment:
Is there a reason why the font sizes here and in the other places for the header are defined in absolute sizes? Would it make sense to define them in relative units to allow for better scaling?

Reply:
We just stay faithful to previous development here.